File: Windows/js/prime3.js
function main() {
    var number;
    var count;
    var divisor;
    var divisorsquare;
    var remainder;
    var isprime;

    console.log(2);
    number = 1;
    for (count = 1; count <= 99; count++) {
        do {
            number = 2 + number;
            divisor = 3;
            divisorsquare = divisor * divisor;
            isprime = true;
            while (isprime && divisorsquare <= number) {
                remainder = number % divisor;
                if (remainder == 0) {
                    isprime = false;
                } else {
                    divisor = 2 + divisor;
                    divisorsquare = divisor * divisor;
                }
            }
            if (isprime) {
                console.log(number);
            }
        } while (!isprime);
    }
}

main();

File: Windows/Python/prime3.py
print(2)
number = 1
for count in range(1, 9 + 1, 1):
    while True:    #This simulates a Do Loop
        number = 2 + number
        divisor = 3
        divisorsquare = divisor * divisor
        isprime = True
        while isprime and divisorsquare <= number:
            remainder = number % divisor
            if remainder == 0:
                isprime = False
            else:
                divisor = 2 + divisor
                divisorsquare = divisor * divisor
        if isprime:
            print(number)
        if isprime: break
